GPU/Common/SoftwareTransformCommon.cpp

```diff
@@ -406,6 +406,8 @@ void SoftwareTransform(
 
 	// Here's the best opportunity to try to detect rectangles used to clear the screen, and
 	// replace them with real clears. This can provide a speedup on certain mobile chips.
+	// Potentially we could also detect things like textured rectangles that cover the screen
+	// like FMV.
```

Comment:
Do you mean so that we can just blit (and filter as needed) rather than render?

-[Unknown]